brax/envs/inverted_pendulum.py

```diff
@@ -46,6 +46,9 @@ class InvertedPendulum(PipelineEnv):
   continuous `(action)` in `[-3, 3]`, where `action` represents the numerical
   force applied to the cart (with magnitude representing the amount of force and
   sign representing the direction)
+  
+  Actions are assumed to be within `[-1, 1]` and are (linearly) scaled 
```

Comment:
I would delete the comment since this is the intended behavior, and we generally try to keep the docstrings in line with gym/gymnasium (granted they decided to nuke v4 docstring). Same comment for other files.